Added STREAMING sources to tigon repro

achalpandey opened the pull request:
On Behalf of @spatsch :
Added all streaming sources to tigon repro with exception of
documentation which should be integrated rather then just copied.

Commit:
Added STREAMING sources to tigon repro

Added all streaming sources to tigon repro with exception of
documentation which should be integrated rather then just copied.

achalpandey commented:
Do we need the views directories under tigon-sql/views and tigon-sql/src/views? I notice that we have these directories in the STREAMING repo as well. But they are empty. 

Apart from that everything looks good. Kindly modify the build scripts to comply to this new directory structure.

spatsch commented:
Views are a supported features. We don’t have any we are currently releasing though.

Oliver

achalpandey commented:
Woops! Can we please move all the files/directories that were added under tigon-sql/src to tigon-sql/src/main/c/

@spatsch does it make sense to add the views folder when we release? we can always create new folders as and when required.

spatsch commented:
fine with me do you want to do it?  Don’t want to generate merge conflicts if multiple people work on the directory structure.

Oliver

achalpandey commented:
I haven't modified  anything on this PR (pull request) to avoid merge
conflicts. As you pointed out,  the simplest way to do so is by having only
one active contributor. So please feel free to make changes to the
directory structure and the build script.

Note : I'll be happy to make these changes,  if you feel that's the best
way to collaborate. :-)

spatsch commented:
I can change it. Do you want all the directories I added in tigon-sql (cfg,views etc..) under src/main/c or just the directories in src ? What about include ?

Oliver

achalpandey commented:
Only the directories under tigon-sql/src should go under src/main/c

So effectively:
#Top level directories in the STREMAING repo like include, bin and cfg go under tigon-sql/ 
#Directories in the STREAMING repo under src go under tigon-sql/src/main/c

Commit:
moved directory to /main/c

Commit:
moving test scripts out

Commit:
rename qlib directory

vshkap committed:
Changed alll build scripts, make files and example to reflect new directory structure.

Commit:
fixed double processing bug

achalpandey commented on tigon-sql/lib/README:
```diff
@@ -0,0 +1 @@
+contains all local libraries
```
Please remove this duplicate README file. This directory now has 2 README files.

achalpandey commented on tigon-sql/src/main/c/externaltar/README:
```diff
@@ -0,0 +1 @@
+Place for auxliary tar files Gigascope might use.
```
Can we please remove this directory if we are not using it. Also please remove references to Gigascope

spatsch replied:
Yes. You want me to do that in the feature/mergewithgs branch ?

Oliver

From: achalpandey <notifications@github.com<mailto:notifications@github.com>>
Reply-To: caskco/tigon <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, September 16, 2014 at 4:54 PM EDT
To: caskco/tigon <tigon@noreply.github.com<mailto:tigon@noreply.github.com>>
Cc: spatsch <os1872@att.com<mailto:os1872@att.com>>
Subject: Re: [tigon] Added STREAMING sources to tigon repro (#3)

In tigon-sql/src/main/c/externaltar/README:

> @@ -0,0 +1 @@
> +Place for auxliary tar files Gigascope might use.

Can we please remove this directory if we are not using it.

—
Reply to this email directly or view it on GitHubhttps://github.com/caskco/tigon/pull/3/files#r17629755.

achalpandey commented:
As discussed please remove the unused views directories at tigon-sql/views & tigon-sql/src/main/c/views.
We can add these directories later, when we have support for views.

achalpandey commented:
Looks like we still have some references to "Gigascope" in our code. Can we please remove these. A grep through our codebase revealed the following usages.

`$ find . -iname "*" -exec grep -n -H -i gigascope {} \; 2>/dev/null`
- ./tigon-sql/include/gslog.h:17: \* gslog.h: syslog wrappers for Gigascope
- ./tigon-sql/src/main/c/cluster_manager/cluster_manager.cc:48:// root directory of gigascope distribution on all the machines
- ./tigon-sql/src/main/c/cluster_manager/cluster_manager.cc:123:    const char *usage_str = "Usage: %s [-B <build directory>] [-C <config directory>] [-G <gigascope directory>] query_file [query_file ...]\n"
- ./tigon-sql/src/main/c/cluster_manager/cluster_manager.cc:127:        "\t[-G] : <gigascope directory> specifies the root of the gigascope distribution on all cluster nodes and cluster manager\n\n"
- ./tigon-sql/src/main/c/cluster_manager/cluster_manager.cc:135:        fprintf(stderr,"CLUSTER_MANAGER ERROR: $HOME is not set - unable to find root directory of gigascope distribution\n");
- ./tigon-sql/src/main/c/externaltar/README:1:Place for auxliary tar files Gigascope might use.
- ./tigon-sql/src/main/c/lib/gscphost/include/gscpipc.h:17: \* the gigascope framework to communicate tuples between different

ted-johnson commented:
I thought the agreement was to keep it.

From: achalpandey [mailto:notifications@github.com]
Sent: Tuesday, September 16, 2014 5:10 PM
To: caskco/tigon
Subject: Re: [tigon] Added STREAMING sources to tigon repro (#3)

As discussed please remove the unused views directories at tigon-sql/views & tigon-sql/src/main/c/views.
We can add these directories later, when we have support for views.

—
Reply to this email directly or view it on GitHubhttps://github.com/caskco/tigon/pull/3#issuecomment-55814083.

achalpandey commented:
As per my last exchange with Oliver, refer the conversation at the top of
the thread https://github.com/caskco/tigon/pull/3 . I was under the
impression that we were removing views for now and will add it whenever we
add support for that feature.

Please correct me if I missed something.

On Tue, Sep 16, 2014 at 2:39 PM, ted-johnson notifications@github.com
wrote:

> I thought the agreement was to keep it.
> 
> From: achalpandey [mailto:notifications@github.com]
> Sent: Tuesday, September 16, 2014 5:10 PM
> To: caskco/tigon
> Subject: Re: [tigon] Added STREAMING sources to tigon repro (#3)
> 
> As discussed please remove the unused views directories at tigon-sql/views
> & tigon-sql/src/main/c/views.
> We can add these directories later, when we have support for views.
> 
> —
> Reply to this email directly or view it on GitHub<
> https://github.com/caskco/tigon/pull/3#issuecomment-55814083>.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/caskco/tigon/pull/3#issuecomment-55817904.

vshkap committed:
Removed remaining Gigascope references

vshkap commented:
I replaced a few surviving Gigascope references that Achal found with Tigon and Tigon-SQL. The only place where it still remains is tigon-sql/src/main/c/externaltar/README which my understanding is going to get removed.

achalpandey commented on tigon-sql/bin/README:
```diff
@@ -0,0 +1 @@
+Contains the ftacompiler and other binaries after compilation
```
Please delete duplicate README

Commit:
removed unneeded directory

spatsch commented:
I thought so too. As it’s in the manual and if somebody wants to try that s/he needs it.

Oliver

From: ted-johnson <notifications@github.com<mailto:notifications@github.com>>
Reply-To: caskco/tigon <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, September 16, 2014 at 5:39 PM EDT
To: caskco/tigon <tigon@noreply.github.com<mailto:tigon@noreply.github.com>>
Cc: spatsch <os1872@att.com<mailto:os1872@att.com>>
Subject: Re: [tigon] Added STREAMING sources to tigon repro (#3)

I thought the agreement was to keep it.

From: achalpandey [mailto:notifications@github.com]
Sent: Tuesday, September 16, 2014 5:10 PM
To: caskco/tigon
Subject: Re: [tigon] Added STREAMING sources to tigon repro (#3)

As discussed please remove the unused views directories at tigon-sql/views & tigon-sql/src/main/c/views.
We can add these directories later, when we have support for views.

—
Reply to this email directly or view it on GitHubhttps://github.com/caskco/tigon/pull/3#issuecomment-55814083.

—
Reply to this email directly or view it on GitHubhttps://github.com/caskco/tigon/pull/3#issuecomment-55817904.

achalpandey commented:
The following files are not cleaned up after I run make clean. Can we please remove all the extra files we don't need. I would imagine that after a make clean we should be back at a clean slate and no extraneous files generated as part of make install are left in the directory.

I got this list by doing a git status after running make install and make clean

```
    bin/gdat2ascii
bin/gen_tuple_access
bin/gsexit
bin/gsgdatprint
bin/gsprintconsole
bin/process_logs
bin/translate_fta
bin/translate_fta.gcov
lib/libclearinghouse.a
lib/libgscpapp.a
lib/libgscpaux.a
lib/libgscphftaaux.a
lib/libgscphost.a
lib/libgscphostaux.a
lib/libgscpinterface.a
lib/libgscplftaaux.a
lib/libgscprts.a
src/main/c/ftacmp/analyze_fta.gcov.gcno
src/main/c/ftacmp/ext_fcns.tab.gcov.gcno
src/main/c/ftacmp/ext_fcnslexer.gcov.gcno
src/main/c/ftacmp/field_list.gcov.gcno
src/main/c/ftacmp/fta.tab.gcov.gcno
src/main/c/ftacmp/ftalexer.gcov.gcno
src/main/c/ftacmp/generate_lfta_code.gcov.gcno
src/main/c/ftacmp/generate_nic_code.gcov.gcno
src/main/c/ftacmp/generate_utils.gcov.gcno
src/main/c/ftacmp/iface_q.gcov.gcno
src/main/c/ftacmp/ifq.tab.gcov.gcno
src/main/c/ftacmp/ifqlexer.gcov.gcno
src/main/c/ftacmp/interfacelib.a
src/main/c/ftacmp/nic.tab.gcov.gcno
src/main/c/ftacmp/nic_def.gcov.gcno
src/main/c/ftacmp/niclexer.gcov.gcno
src/main/c/ftacmp/parse_partn.gcov.gcno
src/main/c/ftacmp/parse_schema.gcov.gcno
src/main/c/ftacmp/partn.tab.gcov.gcno
src/main/c/ftacmp/partnlexer.gcov.gcno
src/main/c/ftacmp/query_plan.gcov.gcno
src/main/c/ftacmp/res.tab.gcov.gcno
src/main/c/ftacmp/reslexer.gcov.gcno
src/main/c/ftacmp/stream_query.gcov.gcno
src/main/c/ftacmp/translate_fta.gcov.gcno
src/main/c/ftacmp/type_objects.gcov.gcno
src/main/c/ftacmp/xml.tab.gcov.gcno
src/main/c/ftacmp/xmllexer.gcov.gcno
src/main/c/tools/gdatcat
src/main/c/tools/gssinksim
```

achalpandey commented:
@spatsch @ted-johnson Ok, lets keep the views folder around, since we speak about it in the document.

ted-johnson commented:
The gcno files are created as part of the code coverage mechanism that was requested.
They are generated by running the test executable, and are cleaned at the start of the test suite execution.

The excutable and library files are cleaned in the directories where they are created.
Make install copies these files from the source directories into the bin and lib directories.

From: achalpandey [mailto:notifications@github.com]
Sent: Wednesday, September 17, 2014 3:35 PM
To: caskco/tigon
Cc: JOHNSON, THEODORE (THEODORE)
Subject: Re: [tigon] Added STREAMING sources to tigon repro (#3)

The following files are not cleaned up after I run make clean. Can we please remove all the extra files we don't need. I would imagine that after a make clean we should be back at a clean slate and no extraneous files generated as part of make install are left in the directory.

I got this list by doing a git status after running make install and make clean

```
bin/gdat2ascii
```

bin/gen_tuple_access

bin/gsexit

bin/gsgdatprint

bin/gsprintconsole

bin/process_logs

bin/translate_fta

bin/translate_fta.gcov

lib/libclearinghouse.a

lib/libgscpapp.a

lib/libgscpaux.a

lib/libgscphftaaux.a

lib/libgscphost.a

lib/libgscphostaux.a

lib/libgscpinterface.a

lib/libgscplftaaux.a

lib/libgscprts.a

src/main/c/ftacmp/analyze_fta.gcov.gcno

src/main/c/ftacmp/ext_fcns.tab.gcov.gcno

src/main/c/ftacmp/ext_fcnslexer.gcov.gcno

src/main/c/ftacmp/field_list.gcov.gcno

src/main/c/ftacmp/fta.tab.gcov.gcno

src/main/c/ftacmp/ftalexer.gcov.gcno

src/main/c/ftacmp/generate_lfta_code.gcov.gcno

src/main/c/ftacmp/generate_nic_code.gcov.gcno

src/main/c/ftacmp/generate_utils.gcov.gcno

src/main/c/ftacmp/iface_q.gcov.gcno

src/main/c/ftacmp/ifq.tab.gcov.gcno

src/main/c/ftacmp/ifqlexer.gcov.gcno

src/main/c/ftacmp/interfacelib.a

src/main/c/ftacmp/nic.tab.gcov.gcno

src/main/c/ftacmp/nic_def.gcov.gcno

src/main/c/ftacmp/niclexer.gcov.gcno

src/main/c/ftacmp/parse_partn.gcov.gcno

src/main/c/ftacmp/parse_schema.gcov.gcno

src/main/c/ftacmp/partn.tab.gcov.gcno

src/main/c/ftacmp/partnlexer.gcov.gcno

src/main/c/ftacmp/query_plan.gcov.gcno

src/main/c/ftacmp/res.tab.gcov.gcno

src/main/c/ftacmp/reslexer.gcov.gcno

src/main/c/ftacmp/stream_query.gcov.gcno

src/main/c/ftacmp/translate_fta.gcov.gcno

src/main/c/ftacmp/type_objects.gcov.gcno

src/main/c/ftacmp/xml.tab.gcov.gcno

src/main/c/ftacmp/xmllexer.gcov.gcno

src/main/c/tools/gdatcat

src/main/c/tools/gssinksim

—
Reply to this email directly or view it on GitHubhttps://github.com/caskco/tigon/pull/3#issuecomment-55947614.

Commit:
Modified REAMDE.md files

Commit:
Merging develop + README changes

achalpandey commented:
@ted-johnson Ok, I have updated the .gitignore file in another PR so that it ignores the generated files.

achalpandey commented:
The following files are missing licenses. Kindly add the required license as per our discussion.

  tigon/tigon-sql/bin/accept_recommendations.bat
  tigon/tigon-sql/bin/buildit
  tigon/tigon-sql/bin/buildit.pl
  tigon/tigon-sql/bin/buildit_test.pl
  tigon/tigon-sql/bin/buildit_with-stats
  tigon/tigon-sql/bin/get_last_trace.pl
  tigon/tigon-sql/bin/monitor_gs.pl
  tigon/tigon-sql/bin/parse_cpuinfo.pl
  tigon/tigon-sql/bin/pin_processes.pl
  tigon/tigon-sql/bin/split_trace.pl
  tigon/tigon-sql/bin/start_processing
  tigon/tigon-sql/build_and_install.sh
  tigon/tigon-sql/cfg/example.ifq
  tigon/tigon-sql/cfg/external_fcns.def
  tigon/tigon-sql/cfg/field_list.xml
  tigon/tigon-sql/cfg/ifres.xml
  tigon/tigon-sql/cfg/internal_fcn.def
  tigon/tigon-sql/cfg/localhost.ifq
  tigon/tigon-sql/cfg/packet_schema.txt
  tigon/tigon-sql/include/hfta/merge_operator_oop.h
  tigon/tigon-sql/qlib/CSVEXAMPLELIB/ex2_src.gsql
  tigon/tigon-sql/src/main/c/cluster_manager/ifq.tab.cc
  tigon/tigon-sql/src/main/c/cluster_manager/ifq.tab.cc.h
  tigon/tigon-sql/src/main/c/cluster_manager/ifqlexer.cc
  tigon/tigon-sql/src/main/c/cluster_manager/Makefile
  tigon/tigon-sql/src/main/c/cluster_manager/res.output
  tigon/tigon-sql/src/main/c/cluster_manager/res.tab.cc
  tigon/tigon-sql/src/main/c/cluster_manager/res.tab.cc.h
  tigon/tigon-sql/src/main/c/cluster_manager/reslexer.cc
  tigon/tigon-sql/src/main/c/ftacmp/ext_fcns.tab.cc
  tigon/tigon-sql/src/main/c/ftacmp/ext_fcns.tab.cc.h
  tigon/tigon-sql/src/main/c/ftacmp/ext_fcnslexer.cc
  tigon/tigon-sql/src/main/c/ftacmp/fta.tab.cc
  tigon/tigon-sql/src/main/c/ftacmp/fta.tab.cc.h
  tigon/tigon-sql/src/main/c/ftacmp/ftalexer.cc
  tigon/tigon-sql/src/main/c/ftacmp/ifq.tab.cc
  tigon/tigon-sql/src/main/c/ftacmp/ifq.tab.cc.h
  tigon/tigon-sql/src/main/c/ftacmp/ifqlexer.cc
  tigon/tigon-sql/src/main/c/ftacmp/Makefile
  tigon/tigon-sql/src/main/c/ftacmp/Makefile.gcov
  tigon/tigon-sql/src/main/c/ftacmp/nic.tab.cc
  tigon/tigon-sql/src/main/c/ftacmp/nic.tab.cc.h
  tigon/tigon-sql/src/main/c/ftacmp/niclexer.cc
  tigon/tigon-sql/src/main/c/ftacmp/partn.tab.cc
  tigon/tigon-sql/src/main/c/ftacmp/partn.tab.cc.h
  tigon/tigon-sql/src/main/c/ftacmp/partnlexer.cc
  tigon/tigon-sql/src/main/c/ftacmp/res.tab.cc
  tigon/tigon-sql/src/main/c/ftacmp/res.tab.cc.h
  tigon/tigon-sql/src/main/c/ftacmp/reslexer.cc
  tigon/tigon-sql/src/main/c/ftacmp/schema.tab.cc
  tigon/tigon-sql/src/main/c/ftacmp/schema.tab.cc.h
  tigon/tigon-sql/src/main/c/ftacmp/schemalexer.cc
  tigon/tigon-sql/src/main/c/ftacmp/xml.tab.cc
  tigon/tigon-sql/src/main/c/ftacmp/xml.tab.hh
  tigon/tigon-sql/src/main/c/ftacmp/xmllexer.cc
  tigon/tigon-sql/src/main/c/lib/gscpapp/Makefile
  tigon/tigon-sql/src/main/c/lib/gscpaux/block_allocator.cpp
  tigon/tigon-sql/src/main/c/lib/gscpaux/block_allocator.h
  tigon/tigon-sql/src/main/c/lib/gscpaux/json.cpp
  tigon/tigon-sql/src/main/c/lib/gscpaux/json.h
  tigon/tigon-sql/src/main/c/lib/gscpaux/Makefile
  tigon/tigon-sql/src/main/c/lib/gscphftaaux/Makefile
  tigon/tigon-sql/src/main/c/lib/gscphost/Makefile
  tigon/tigon-sql/src/main/c/lib/gscphostaux/Makefile
  tigon/tigon-sql/src/main/c/lib/gscplftaaux/Makefile
  tigon/tigon-sql/src/main/c/lib/gscprts/Makefile
  tigon/tigon-sql/src/main/c/lib/Makefile
  tigon/tigon-sql/src/main/c/Makefile
  tigon/tigon-sql/src/main/c/tools/Makefile
  tigon/tigon-sql/src/main/c/tools/xml.l
  tigon/tigon-sql/src/main/c/tools/xml.tab.cc
  tigon/tigon-sql/src/main/c/tools/xml.tab.hh
  tigon/tigon-sql/src/main/c/tools/xml.y
  tigon/tigon-sql/src/main/c/tools/xmllexer.cc
  tigon/tigon-sql/src/test/scripts/clean_coverage_files.pl
  tigon/tigon-sql/src/test/scripts/extract_coverage_results.pl
  tigon/tigon-sql/src/test/scripts/run_test.pl
  tigon/tigon-sql/src/test/scripts/TestAggregates/bool/Expected_type_bool.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/bool/Input.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/bool/type_bool_1.gsql
  tigon/tigon-sql/src/test/scripts/TestAggregates/float/Expected_type_float_1.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/float/Input.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/float/type_float_1.gsql
  tigon/tigon-sql/src/test/scripts/TestAggregates/int/Expected_type_int_1.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/int/Input.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/int/type_int_1.gsql
  tigon/tigon-sql/src/test/scripts/TestAggregates/IP/Expected_type_IP_1.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/IP/Input.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/IP/type_IP_1.gsql
  tigon/tigon-sql/src/test/scripts/TestAggregates/IPV6/Expected_type_IPV6_1.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/IPV6/Input.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/IPV6/type_IPV6_1.gsql
  tigon/tigon-sql/src/test/scripts/TestAggregates/llong/Expected_type_llong_1.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/llong/Input.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/llong/type_llong_1.gsql
  tigon/tigon-sql/src/test/scripts/TestAggregates/packet_schema_test.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/run_test.pl
  tigon/tigon-sql/src/test/scripts/TestAggregates/string/Expected_type_string_1.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/string/Input.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/string/type_string_1.gsql
  tigon/tigon-sql/src/test/scripts/TestAggregates/uint/Expected_type_uint_1.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/uint/Input.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/uint/type_uint_1.gsql
  tigon/tigon-sql/src/test/scripts/TestAggregates/ullong/Expected_type_ullong_1.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/ullong/Input.txt
  tigon/tigon-sql/src/test/scripts/TestAggregates/ullong/type_ullong_1.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/complex_A.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/complex_B.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/Expected.txt
  tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/Input.txt
  tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/JoinQuery.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/complex_A.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/complex_B.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/Expected.txt
  tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/Input.txt
  tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/JoinQuery.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/complex_A.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/complex_B.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/Expected.txt
  tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/Input.txt
  tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/JoinQuery.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/complex_A.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/complex_B.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/Expected.txt
  tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/Input.txt
  tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/JoinQuery.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/packet_schema_test.txt
  tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/complex_A.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/complex_B.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/Expected.txt
  tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/Input.txt
  tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/JoinQuery.gsql
  tigon/tigon-sql/src/test/scripts/TestJoins/run_test.pl
  tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/equal_query1.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/equal_query2.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/equal_query3.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Expected_equal_query1.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Expected_equal_query2.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Expected_equal_query3.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Input.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Expected_greater_or_equal_query1.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Expected_greater_or_equal_query2.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Expected_greater_or_equal_query3.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/greater_or_equal_query1.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/greater_or_equal_query2.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/greater_or_equal_query3.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Input.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Expected_greater_than_query1.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Expected_greater_than_query2.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Expected_greater_than_query3.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/greater_than_query1.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/greater_than_query2.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/greater_than_query3.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Input.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Expected_in_enumeration_query1.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Expected_in_enumeration_query2.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Expected_in_enumeration_query3.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/in_enumeration_query1.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/in_enumeration_query2.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/in_enumeration_query3.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Input.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/Input.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Expected_not_equal_query1.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Expected_not_equal_query2.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Expected_not_equal_query3.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Input.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/not_equal_query1.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/not_equal_query2.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/not_equal_query3.gsql
  tigon/tigon-sql/src/test/scripts/TestPredicates/packet_schema_test.txt
  tigon/tigon-sql/src/test/scripts/TestPredicates/run_test.pl
  tigon/tigon-sql/src/test/scripts/TestTypes/bool/Input.txt
  tigon/tigon-sql/src/test/scripts/TestTypes/bool/type_bool.gsql
  tigon/tigon-sql/src/test/scripts/TestTypes/float/Input.txt
  tigon/tigon-sql/src/test/scripts/TestTypes/float/type_float.gsql
  tigon/tigon-sql/src/test/scripts/TestTypes/int/Input.txt
  tigon/tigon-sql/src/test/scripts/TestTypes/int/type_int.gsql
  tigon/tigon-sql/src/test/scripts/TestTypes/IP/Input.txt
  tigon/tigon-sql/src/test/scripts/TestTypes/IP/type_IP.gsql
  tigon/tigon-sql/src/test/scripts/TestTypes/IPV6/Input.txt
  tigon/tigon-sql/src/test/scripts/TestTypes/IPV6/type_IPV6.gsql
  tigon/tigon-sql/src/test/scripts/TestTypes/llong/Input.txt
  tigon/tigon-sql/src/test/scripts/TestTypes/llong/type_llong.gsql
  tigon/tigon-sql/src/test/scripts/TestTypes/packet_schema_test.txt
  tigon/tigon-sql/src/test/scripts/TestTypes/run_test.pl
  tigon/tigon-sql/src/test/scripts/TestTypes/string/Input.txt
  tigon/tigon-sql/src/test/scripts/TestTypes/string/type_string.gsql
  tigon/tigon-sql/src/test/scripts/TestTypes/uint/Input.txt
  tigon/tigon-sql/src/test/scripts/TestTypes/uint/type_uint.gsql
  tigon/tigon-sql/src/test/scripts/TestTypes/ullong/Input.txt
  tigon/tigon-sql/src/test/scripts/TestTypes/ullong/type_ullong.gsql

achalpandey commented:
I have removed the duplicate README files and merged in the new changes from develop. Please fix the license issue and then we should be good to merge.

Please do a "git pull" to get all the new changes.

Commit:
Added regex matching code

As Linux does not support regnexec replaces regnexec with regexec. This
costs an extra copy and limits the regex functions to strings with no 0
within the string.

Commit:
fixed str end bug

achalpandey commented:
Please update https://github.com/caskco/tigon/blob/feature/mergewithgs/tigon-sql/src/test/scripts/README.md to reflect new directory structure. It starts with "To run the test suite, cd to Streaming/test and execute run_test.pl "

achalpandey commented:
Looks like we also need to update https://github.com/caskco/tigon/blob/feature/mergewithgs/tigon-sql/src/test/scripts/run_test.pl

Currently the script fails with the following error message

`
Could not locate STREAMING directory.
Use of uninitialized value within %Months in concatenation (.) or string at run_test.pl line 158.
Use of uninitialized value $LogFile in concatenation (.) or string at run_test.pl line 161.
Use of uninitialized value $LogFile in concatenation (.) or string at run_test.pl line 161.
Could not open the log file  for write.
`

Commit:
Added initiation of SQL library build & test through maven

vshkap commented:
I took a pass and added licenses to all the files where it made sense. Below are the files that are not going to have licenses attached:

Flex/bison generated code:

tigon/tigon-sql/src/main/c/cluster_manager/ifq.tab.cc

tigon/tigon-sql/src/main/c/cluster_manager/ifq.tab.cc.h

tigon/tigon-sql/src/main/c/cluster_manager/ifqlexer.cc

tigon/tigon-sql/src/main/c/cluster_manager/res.tab.cc

tigon/tigon-sql/src/main/c/cluster_manager/res.tab.cc.h

tigon/tigon-sql/src/main/c/cluster_manager/reslexer.cc

tigon/tigon-sql/src/main/c/ftacmp/ext_fcns.tab.cc

tigon/tigon-sql/src/main/c/ftacmp/ext_fcns.tab.cc.h

tigon/tigon-sql/src/main/c/ftacmp/ext_fcnslexer.cc

tigon/tigon-sql/src/main/c/ftacmp/fta.tab.cc

tigon/tigon-sql/src/main/c/ftacmp/fta.tab.cc.h

tigon/tigon-sql/src/main/c/ftacmp/ftalexer.cc

tigon/tigon-sql/src/main/c/ftacmp/ifq.tab.cc

tigon/tigon-sql/src/main/c/ftacmp/ifq.tab.cc.h

tigon/tigon-sql/src/main/c/ftacmp/ifqlexer.cc

tigon/tigon-sql/src/main/c/ftacmp/nic.tab.cc

tigon/tigon-sql/src/main/c/ftacmp/nic.tab.cc.h

tigon/tigon-sql/src/main/c/ftacmp/niclexer.cc

tigon/tigon-sql/src/main/c/ftacmp/partn.tab.cc

tigon/tigon-sql/src/main/c/ftacmp/partn.tab.cc.h

tigon/tigon-sql/src/main/c/ftacmp/partnlexer.cc

tigon/tigon-sql/src/main/c/ftacmp/res.tab.cc

tigon/tigon-sql/src/main/c/ftacmp/res.tab.cc.h

tigon/tigon-sql/src/main/c/ftacmp/reslexer.cc

tigon/tigon-sql/src/main/c/ftacmp/schema.tab.cc

tigon/tigon-sql/src/main/c/ftacmp/schema.tab.cc.h

tigon/tigon-sql/src/main/c/ftacmp/schemalexer.cc

tigon/tigon-sql/src/main/c/ftacmp/xml.tab.cc

tigon/tigon-sql/src/main/c/ftacmp/xml.tab.hh

tigon/tigon-sql/src/main/c/ftacmp/xmllexer.cc

tigon/tigon-sql/src/main/c/tools/xml.tab.cc

tigon/tigon-sql/src/main/c/tools/xml.tab.hh

tigon/tigon-sql/src/main/c/tools/xmllexer.cc

JSON parser distributed under MIT license:

tigon/tigon-sql/src/main/c/lib/gscpaux/block_allocator.cpp

tigon/tigon-sql/src/main/c/lib/gscpaux/block_allocator.h

tigon/tigon-sql/src/main/c/lib/gscpaux/json.cpp

tigon/tigon-sql/src/main/c/lib/gscpaux/json.h

Configuration files:

tigon/tigon-sql/cfg/example.ifq

tigon/tigon-sql/cfg/localhost.ifq

tigon/tigon-sql/cfg/ifres.xml

tigon/tigon-sql/cfg/external_fcns.def

tigon/tigon-sql/cfg/field_list.xml

tigon/tigon-sql/cfg/internal_fcn.def

tigon/tigon-sql/cfg/packet_schema.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/packet_schema_test.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/packet_schema_test.txt

tigon/tigon-sql/src/test/scripts/TestJoins/packet_schema_test.txt

tigon/tigon-sql/src/test/scripts/TestTypes/packet_schema_test.txt

Input/output data files for test suite:

tigon/tigon-sql/src/test/scripts/TestAggregates/bool/Expected_type_bool.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/bool/Input.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/float/Expected_type_float_1.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/float/Input.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/int/Expected_type_int_1.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/int/Input.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/IP/Expected_type_IP_1.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/IP/Input.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/IPV6/Expected_type_IPV6_1.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/IPV6/Input.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/llong/Expected_type_llong_1.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/llong/Input.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/string/Expected_type_string_1.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/string/Input.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/uint/Expected_type_uint_1.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/uint/Input.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/ullong/Expected_type_ullong_1.txt

tigon/tigon-sql/src/test/scripts/TestAggregates/ullong/Input.txt

tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/Expected.txt

tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/Input.txt

tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/Expected.txt

tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/Input.txt

tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/Expected.txt

tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/Input.txt

tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/Expected.txt

tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/Input.txt

tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/Expected.txt

tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/Input.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Expected_equal_query1.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Expected_equal_query2.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Expected_equal_query3.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Input.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Expected_greater_or_equal_query1.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Expected_greater_or_equal_query2.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Expected_greater_or_equal_query3.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Input.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Expected_greater_than_query1.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Expected_greater_than_query2.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Expected_greater_than_query3.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Input.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Expected_in_enumeration_query1.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Expected_in_enumeration_query2.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Expected_in_enumeration_query3.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Input.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/Input.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Expected_not_equal_query1.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Expected_not_equal_query2.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Expected_not_equal_query3.txt

tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Input.txt

tigon/tigon-sql/src/test/scripts/TestTypes/bool/Input.txt

tigon/tigon-sql/src/test/scripts/TestTypes/float/Input.txt

tigon/tigon-sql/src/test/scripts/TestTypes/int/Input.txt

tigon/tigon-sql/src/test/scripts/TestTypes/IP/Input.txt

tigon/tigon-sql/src/test/scripts/TestTypes/IPV6/Input.txt

tigon/tigon-sql/src/test/scripts/TestTypes/llong/Input.txt

tigon/tigon-sql/src/test/scripts/TestTypes/string/Input.txt

tigon/tigon-sql/src/test/scripts/TestTypes/uint/Input.txt

tigon/tigon-sql/src/test/scripts/TestTypes/ullong/Input.txt

Vlad

From: achalpandey [mailto:notifications@github.com] 
Sent: Wednesday, September 17, 2014 4:53 PM
To: caskco/tigon
Cc: vshkap
Subject: Re: [tigon] Added STREAMING sources to tigon repro (#3)

The following files are missing licenses. Kindly add the required license as per our discussion.

tigon/tigon-sql/bin/accept_recommendations.bat
tigon/tigon-sql/bin/buildit
tigon/tigon-sql/bin/buildit.pl
tigon/tigon-sql/bin/buildit_test.pl
tigon/tigon-sql/bin/buildit_with-stats
tigon/tigon-sql/bin/get_last_trace.pl
tigon/tigon-sql/bin/monitor_gs.pl
tigon/tigon-sql/bin/parse_cpuinfo.pl
tigon/tigon-sql/bin/pin_processes.pl
tigon/tigon-sql/bin/split_trace.pl
tigon/tigon-sql/bin/start_processing
tigon/tigon-sql/build_and_install.sh
tigon/tigon-sql/cfg/example.ifq
tigon/tigon-sql/cfg/external_fcns.def
tigon/tigon-sql/cfg/field_list.xml
tigon/tigon-sql/cfg/ifres.xml
tigon/tigon-sql/cfg/internal_fcn.def
tigon/tigon-sql/cfg/localhost.ifq
tigon/tigon-sql/cfg/packet_schema.txt
tigon/tigon-sql/include/hfta/merge_operator_oop.h
tigon/tigon-sql/qlib/CSVEXAMPLELIB/ex2_src.gsql
tigon/tigon-sql/src/main/c/cluster_manager/ifq.tab.cc
tigon/tigon-sql/src/main/c/cluster_manager/ifq.tab.cc.h
tigon/tigon-sql/src/main/c/cluster_manager/ifqlexer.cc
tigon/tigon-sql/src/main/c/cluster_manager/Makefile
tigon/tigon-sql/src/main/c/cluster_manager/res.output
tigon/tigon-sql/src/main/c/cluster_manager/res.tab.cc
tigon/tigon-sql/src/main/c/cluster_manager/res.tab.cc.h
tigon/tigon-sql/src/main/c/cluster_manager/reslexer.cc
tigon/tigon-sql/src/main/c/ftacmp/ext_fcns.tab.cc
tigon/tigon-sql/src/main/c/ftacmp/ext_fcns.tab.cc.h
tigon/tigon-sql/src/main/c/ftacmp/ext_fcnslexer.cc
tigon/tigon-sql/src/main/c/ftacmp/fta.tab.cc
tigon/tigon-sql/src/main/c/ftacmp/fta.tab.cc.h
tigon/tigon-sql/src/main/c/ftacmp/ftalexer.cc
tigon/tigon-sql/src/main/c/ftacmp/ifq.tab.cc
tigon/tigon-sql/src/main/c/ftacmp/ifq.tab.cc.h
tigon/tigon-sql/src/main/c/ftacmp/ifqlexer.cc
tigon/tigon-sql/src/main/c/ftacmp/Makefile
tigon/tigon-sql/src/main/c/ftacmp/Makefile.gcov
tigon/tigon-sql/src/main/c/ftacmp/nic.tab.cc
tigon/tigon-sql/src/main/c/ftacmp/nic.tab.cc.h
tigon/tigon-sql/src/main/c/ftacmp/niclexer.cc
tigon/tigon-sql/src/main/c/ftacmp/partn.tab.cc
tigon/tigon-sql/src/main/c/ftacmp/partn.tab.cc.h
tigon/tigon-sql/src/main/c/ftacmp/partnlexer.cc
tigon/tigon-sql/src/main/c/ftacmp/res.tab.cc
tigon/tigon-sql/src/main/c/ftacmp/res.tab.cc.h
tigon/tigon-sql/src/main/c/ftacmp/reslexer.cc
tigon/tigon-sql/src/main/c/ftacmp/schema.tab.cc
tigon/tigon-sql/src/main/c/ftacmp/schema.tab.cc.h
tigon/tigon-sql/src/main/c/ftacmp/schemalexer.cc
tigon/tigon-sql/src/main/c/ftacmp/xml.tab.cc
tigon/tigon-sql/src/main/c/ftacmp/xml.tab.hh
tigon/tigon-sql/src/main/c/ftacmp/xmllexer.cc
tigon/tigon-sql/src/main/c/lib/gscpapp/Makefile
tigon/tigon-sql/src/main/c/lib/gscpaux/block_allocator.cpp
tigon/tigon-sql/src/main/c/lib/gscpaux/block_allocator.h
tigon/tigon-sql/src/main/c/lib/gscpaux/json.cpp
tigon/tigon-sql/src/main/c/lib/gscpaux/json.h
tigon/tigon-sql/src/main/c/lib/gscpaux/Makefile
tigon/tigon-sql/src/main/c/lib/gscphftaaux/Makefile
tigon/tigon-sql/src/main/c/lib/gscphost/Makefile
tigon/tigon-sql/src/main/c/lib/gscphostaux/Makefile
tigon/tigon-sql/src/main/c/lib/gscplftaaux/Makefile
tigon/tigon-sql/src/main/c/lib/gscprts/Makefile
tigon/tigon-sql/src/main/c/lib/Makefile
tigon/tigon-sql/src/main/c/Makefile
tigon/tigon-sql/src/main/c/tools/Makefile
tigon/tigon-sql/src/main/c/tools/xml.l
tigon/tigon-sql/src/main/c/tools/xml.tab.cc
tigon/tigon-sql/src/main/c/tools/xml.tab.hh
tigon/tigon-sql/src/main/c/tools/xml.y
tigon/tigon-sql/src/main/c/tools/xmllexer.cc
tigon/tigon-sql/src/test/scripts/clean_coverage_files.pl
tigon/tigon-sql/src/test/scripts/extract_coverage_results.pl
tigon/tigon-sql/src/test/scripts/run_test.pl
tigon/tigon-sql/src/test/scripts/TestAggregates/bool/Expected_type_bool.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/bool/Input.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/bool/type_bool_1.gsql
tigon/tigon-sql/src/test/scripts/TestAggregates/float/Expected_type_float_1.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/float/Input.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/float/type_float_1.gsql
tigon/tigon-sql/src/test/scripts/TestAggregates/int/Expected_type_int_1.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/int/Input.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/int/type_int_1.gsql
tigon/tigon-sql/src/test/scripts/TestAggregates/IP/Expected_type_IP_1.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/IP/Input.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/IP/type_IP_1.gsql
tigon/tigon-sql/src/test/scripts/TestAggregates/IPV6/Expected_type_IPV6_1.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/IPV6/Input.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/IPV6/type_IPV6_1.gsql
tigon/tigon-sql/src/test/scripts/TestAggregates/llong/Expected_type_llong_1.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/llong/Input.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/llong/type_llong_1.gsql
tigon/tigon-sql/src/test/scripts/TestAggregates/packet_schema_test.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/run_test.pl
tigon/tigon-sql/src/test/scripts/TestAggregates/string/Expected_type_string_1.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/string/Input.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/string/type_string_1.gsql
tigon/tigon-sql/src/test/scripts/TestAggregates/uint/Expected_type_uint_1.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/uint/Input.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/uint/type_uint_1.gsql
tigon/tigon-sql/src/test/scripts/TestAggregates/ullong/Expected_type_ullong_1.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/ullong/Input.txt
tigon/tigon-sql/src/test/scripts/TestAggregates/ullong/type_ullong_1.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/complex_A.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/complex_B.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/Expected.txt
tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/Input.txt
tigon/tigon-sql/src/test/scripts/TestJoins/inner_join/JoinQuery.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/complex_A.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/complex_B.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/Expected.txt
tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/Input.txt
tigon/tigon-sql/src/test/scripts/TestJoins/left_outer_join/JoinQuery.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/complex_A.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/complex_B.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/Expected.txt
tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/Input.txt
tigon/tigon-sql/src/test/scripts/TestJoins/no_predicate_join/JoinQuery.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/complex_A.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/complex_B.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/Expected.txt
tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/Input.txt
tigon/tigon-sql/src/test/scripts/TestJoins/outer_join/JoinQuery.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/packet_schema_test.txt
tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/complex_A.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/complex_B.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/Expected.txt
tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/Input.txt
tigon/tigon-sql/src/test/scripts/TestJoins/right_outer_join/JoinQuery.gsql
tigon/tigon-sql/src/test/scripts/TestJoins/run_test.pl
tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/equal_query1.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/equal_query2.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/equal_query3.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Expected_equal_query1.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Expected_equal_query2.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Expected_equal_query3.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/Equal/Input.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Expected_greater_or_equal_query1.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Expected_greater_or_equal_query2.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Expected_greater_or_equal_query3.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/greater_or_equal_query1.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/greater_or_equal_query2.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/greater_or_equal_query3.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterOrEqual/Input.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Expected_greater_than_query1.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Expected_greater_than_query2.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Expected_greater_than_query3.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/greater_than_query1.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/greater_than_query2.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/greater_than_query3.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/GreaterThan/Input.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Expected_in_enumeration_query1.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Expected_in_enumeration_query2.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Expected_in_enumeration_query3.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/in_enumeration_query1.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/in_enumeration_query2.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/in_enumeration_query3.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/InEnumeration/Input.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/Input.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Expected_not_equal_query1.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Expected_not_equal_query2.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Expected_not_equal_query3.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/Input.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/not_equal_query1.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/not_equal_query2.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/NotEqual/not_equal_query3.gsql
tigon/tigon-sql/src/test/scripts/TestPredicates/packet_schema_test.txt
tigon/tigon-sql/src/test/scripts/TestPredicates/run_test.pl
tigon/tigon-sql/src/test/scripts/TestTypes/bool/Input.txt
tigon/tigon-sql/src/test/scripts/TestTypes/bool/type_bool.gsql
tigon/tigon-sql/src/test/scripts/TestTypes/float/Input.txt
tigon/tigon-sql/src/test/scripts/TestTypes/float/type_float.gsql
tigon/tigon-sql/src/test/scripts/TestTypes/int/Input.txt
tigon/tigon-sql/src/test/scripts/TestTypes/int/type_int.gsql
tigon/tigon-sql/src/test/scripts/TestTypes/IP/Input.txt
tigon/tigon-sql/src/test/scripts/TestTypes/IP/type_IP.gsql
tigon/tigon-sql/src/test/scripts/TestTypes/IPV6/Input.txt
tigon/tigon-sql/src/test/scripts/TestTypes/IPV6/type_IPV6.gsql
tigon/tigon-sql/src/test/scripts/TestTypes/llong/Input.txt
tigon/tigon-sql/src/test/scripts/TestTypes/llong/type_llong.gsql
tigon/tigon-sql/src/test/scripts/TestTypes/packet_schema_test.txt
tigon/tigon-sql/src/test/scripts/TestTypes/run_test.pl
tigon/tigon-sql/src/test/scripts/TestTypes/string/Input.txt
tigon/tigon-sql/src/test/scripts/TestTypes/string/type_string.gsql
tigon/tigon-sql/src/test/scripts/TestTypes/uint/Input.txt
tigon/tigon-sql/src/test/scripts/TestTypes/uint/type_uint.gsql
tigon/tigon-sql/src/test/scripts/TestTypes/ullong/Input.txt
tigon/tigon-sql/src/test/scripts/TestTypes/ullong/type_ullong.gsql

—
Reply to this email directly or view it on GitHub https://github.com/caskco/tigon/pull/3#issuecomment-55958299 .Image removed by sender.

vshkap committed:
Added missing licenses

Commit:
ensure that count(*) returns int.

Commit:
Merge branch 'feature/mergewithgs' of https://github.com/caskco/tigon into feature/mergewithgs

Commit:
Increase sleap time in runit to ensure gshub has time to create its log file

vshkap committed:
Added missing return value for some functors

vshkap committed:
Fixed typos introduced in the last commit

Commit:
Added license check exclusions

Commit:
Pulled develop

achalpandey commented:
@vshkap Thank you for adding the licenses. I have made changes to maven to ignore all the files listed in the exclusion list. 
@all I pulled in the new changes from the develop branch and the branch can now be merged automatically.

The only remaining items are:
-  Fix the `run_test.pl` script, to work with the new directory structure
-  Update the test/scripts/README.md file and replace "STREAMING/test" with "tigon/tigon-sql/src/test/scripts"

We are almost there! Lets close out this PR by end of day today! :smiley: 

Note: Don't forget to do a `git pull` to get all the changes

Commit:
try to fix the test scripts
